lighthouse-core/gather/gatherers/dobetterweb/domstats.js

```diff
@@ -148,8 +148,8 @@ class DOMStats extends Gatherer {
     })()`;
     return passContext.driver.sendCommand('DOM.enable')
       .then(() => passContext.driver.evaluateAsync(expression, {useIsolation: true}))
```

Comment:
we'll probably want to piggyback on our `getDOMStats` function to compute the total body nodes

Reply:
Yeah, sounds reasonable to do calculation right there :)